modules/items.py

```diff
@@ -6,16 +6,18 @@ class Item():
 
     This is the base class for items
     """
-    def __init__(self, name):
+    def __init__(self, name=None, description=None):
         self.name = name
+        self.description = description
 
-class Bag():
+class Bag(Item):
```

Comment:
Curious that you made the bag an item.  I wasn't thinking of it as one, but sure why not :)